reference/requirements.md

```diff
@@ -44,7 +44,6 @@ For external access to the Anbox Cloud Appliance, you must expose a couple of ne
 
 The Anbox Cloud Appliance supports the following Ubuntu versions:
 
-* Ubuntu 20.04 (Focal Fossa)
```

Comment:
Technically, our old application still supports running on 20.04 given that we will still roll out old appliance(epoch=0) in 1.24 release .

Reply:
Hmm, but according to our [deprecation notices](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/deprecation-notices/#ubuntu-20-04-focal), we are supposed to drop 20.04 support from 1.24.0. 

Yes the appliance epoch=0 will be around until 1.25.0 but I think we can start dropping official support for focal. WDYT @morphis ?

Reply:
@adglkh technically a lot is possible :-) However from a support/maintenance perspective all support for focal is gone by now.

Reply:
Okay, let's drop that